fili-core/src/main/java/com/yahoo/bard/webservice/web/responseprocessors/PartialDataV2ResponseProcessor.java

```diff
@@ -0,0 +1,100 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web.responseprocessors;
+
+import com.yahoo.bard.webservice.druid.client.FailureCallback;
+import com.yahoo.bard.webservice.druid.client.HttpErrorCallback;
+import com.yahoo.bard.webservice.druid.model.query.DruidAggregationQuery;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Response processor for partial data V2.
+ * <p>
+ * If Fili expects data that is not returned from Druid, it returns an error.
+ */
+public class PartialDataV2ResponseProcessor implements FullResponseProcessor {
+
+    private static final Logger LOG = LoggerFactory.getLogger(PartialDataV2ResponseProcessor.class);
+
+    private final ResponseProcessor next;
+
+    /**
+     * Constructor.
+     *
+     * @param next  Next ResponseProcessor in the chain
+     */
+    public PartialDataV2ResponseProcessor(ResponseProcessor next) {
+        this.next = next;
+    }
+
+    @Override
+    public ResponseContext getResponseContext() {
+        return next.getResponseContext();
+    }
+
+    @Override
+    public FailureCallback getFailureCallback(DruidAggregationQuery<?> druidQuery) {
+        return next.getFailureCallback(druidQuery);
+    }
+
+    @Override
+    public HttpErrorCallback getErrorCallback(DruidAggregationQuery<?> druidQuery) {
+        return next.getErrorCallback(druidQuery);
+    }
+
+    @Override
+    public void processResponse(JsonNode json, DruidAggregationQuery<?> query, LoggingContext metadata) {
+        validateJsonResponse(json);
+
+        int statusCode = json.get("status-code").asInt();
+        if (statusCode == 200) {
+            // implementation is blocked by https://github.com/yahoo/fili/pull/262
+        }
+
+        next.processResponse(json, query, metadata);
+    }
+
+    /**
+     * Validates JSON response object to make sure it contains all of the following information.
+     * <ul>
+     *     <li>X-Druid-Response-Context</li>
+     *     <ol>
```

Comment:
I don't think these list items are nesting the way you intend them to, so I don't think it will render correctly. I _think_ the closing `</li>` with the header name on it should go _after_ the closing `</ol>` for this list.